package.json

```diff
@@ -8,7 +8,7 @@
   "license": "MIT",
   "scripts": {
     "test": "jest",
-    "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook": "start-storybook -p 9001 -c .storybook -s ./.storybook/static",
```

Comment:
Static assets should just go inside the root `assets` dir, which I had added to the `.gitignore` file so you’ll have to remove that.